rllib/evaluation/collectors/simple_list_collector.py

```diff
@@ -119,28 +126,28 @@ def build(self, view_requirements: Dict[str, ViewRequirement]) -> \
                 requirements dict needed to build the SampleBatch from the raw
                 buffers (which may have data shifts as well as mappings from
                 view-col to data-col in them).
+            inference_view_requirements (Dict[str, ViewRequirement]: The view
+                requirements dict needed to build an input dict for a ModelV2
+                forward call.
```

Comment:
This argument doesn't seem to be used, can we remove it?

Reply:
done.